x-pack/legacy/plugins/code/public/components/admin_page/setup_guide.tsx

```diff
@@ -23,80 +25,149 @@ import { RootState } from '../../reducers';
 
 const steps = [
   {
-    title: 'Check if multiple Kibana instances are used as a cluster',
+    title: i18n.translate('xpack.code.adminPage.setupGuide.checkMultiInstanceTitle', {
+      defaultMessage: 'Check if multiple Kibana instances are used as a clusterURL',
+    }),
     children: (
       <EuiText>
-        <p>If you are using single Kibana instance, you can skip this step.</p>
         <p>
-          If you are using multiple Kibana instances, you need to assign one Kibana instance as
-          `Code node`. To do this, add the following line of code into your kibana.yml file of every
-          Kibana instance and restart the instances:
+          <FormattedMessage
+            id="xpack.code.adminPage.setupGuide.checkMultiInstanceDescription1"
```

Comment:
I think it would be better to use a more descriptive ID here, like `...setupGuide.skipOnSingleInstanceDescription`

(Same for the translations following this pattern below)

Reply:
my concern here are
1. the text is subject to change
2. because the json files are sort alphabetically by key, that will makes the reading in translation pretty hard, I guess that why I see similar naming in other product as well? e.g.
![image](https://user-images.githubusercontent.com/1646270/61924199-5a383180-af99-11e9-90f3-2f2378a3bf69.png)